app/frontend/wallet/max-amount-calculator.ts

```diff
@@ -24,15 +26,27 @@ export const MaxAmountCalculator = () => {
         profitableInputs.map(({tokenBundle}) => tokenBundle)
       )
       const inputBalance = getInputBalance(profitableInputs)
-      const additionalLovelaceAmount =
+
+      const changeOutputs = splitTxOutput(
+        {
+          isChange: false,
+          address,
+          coins: computeMinUTxOLovelaceAmount(inputsTokenBundle) as Lovelace,
```

Comment:
is there any point in passing a computed amount of coins if splitTxOutput overwrites the amounts?